ranger/config/rifle.conf

```diff
@@ -106,6 +106,7 @@ ext rb  = ruby -- "$1"
 ext js  = node -- "$1"
 ext sh  = sh -- "$1"
 ext php = php -- "$1"
+ext dart = dart "$1"
```

Comment:
Does dart support the `--` convention?

Reply:
just had a look on dart's documentation, found out yes it does support it.  Lemme update the pr accordingly :smile: